legacy/src/api/merchants/merchants.md

```diff
@@ -29,24 +29,38 @@ which define the payment methods available for a Payment Request.
+| id               | String             | Merchant's unique identifier.                                                                               |
+| accountId        | String             | Id of Merchant's owning Centrapay account.                                                                  |
+| name             | String             | Merchant name.                                                                                              |
+| country          | String             | Merchant [ISO 3166]{:.external} country code. Must match the "region" on the [Account][].                   |
+| createdAt        | {% dt Timestamp %} | When the Merchant was created.                                                                              |
+| createdBy        | {% dt CRN %}       | The User or API Key that created the Merchant.                                                              |
+| updatedAt        | {% dt Timestamp %} | When the Merchant was updated.                                                                              |
+| updatedBy        | {% dt CRN %}       | The User or API Key that updated the Merchant.                                                              |
+| onboardingStatus | String             | The onboarding status of the Merchant. See [Onboarding Statuses](#onboarding-statuses) for possible values. |
 
 {% h4 Optional Fields %}
 
-|      Field       |                  Type                   |                        Description                        |
-| :--------------- | :-------------------------------------- | :-------------------------------------------------------- |
-| test             | Boolean                                 | **EXPERIMENTAL** Flag indicating merchant is for testing. |
-| settlementConfig | [Settlement Config](#settlement-config) | **EXPERIMENTAL** Merchant settlement config               |
-| location         | {% dt Location %}                       | **EXPERIMENTAL** Physical Location of Merchant            |
+|         Field          |                  Type                   |                                Description                                |
+| :--------------------- | :-------------------------------------- | :------------------------------------------------------------------------ |
+| test                   | Boolean                                 | **EXPERIMENTAL** Flag indicating merchant is for testing.                 |
+| settlementConfig       | [Settlement Config](#settlement-config) | **EXPERIMENTAL** Merchant settlement config.                               |
+| location               | {% dt Location %}                       | **EXPERIMENTAL** Physical Location of Merchant.                            |
+| onboardingStatusReason | String                                  | The reason associated with the [Onboarding Status](#onboarding-statuses). |
```

Comment:
We should have descriptions for these values

Reply:
Do you mean a separate table with each possible `onboardingStatusReason` with a relevant description?

Reply:
Yep - that's what I was thinking

Reply:
I've added a table with each reason and placeholder descriptions - open to modifications. I wasn't sure on a couple of the descriptions e.g. `centrapay-discontinued` - was there a notion doc somewhere with more details on each of these? I couldn't find it

Reply:
<img width="786" alt="image" src="https://user-images.githubusercontent.com/70119888/211232485-4eb6bddf-5e4a-462d-9f5c-91eec798be61.png">